account_budget_oca/models/account_budget.py

```diff
@@ -0,0 +1,233 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+from_string = fields.Datetime.from_string
+
+
+# ---------------------------------------------------------
+# Budgets
+# ---------------------------------------------------------
+class AccountBudgetPost(models.Model):
+    _name = "account.budget.post"
+    _order = "name"
+    _description = "Budgetary Position"
+
+    name = fields.Char('Name', required=True)
+    account_ids = fields.Many2many(
+        comodel_name='account.account', relation='account_budget_rel',
+        column1='budget_id', column2='account_id', string='Accounts',
+        domain=[('deprecated', '=', False)])
+    crossovered_budget_line_ids = fields.One2many(
```

Comment:
Field `crossovered_budget_line_ids` should be removed here since it's not used. The inverse_name `general_budget_id` in the budget lines is referring to `crossovered.budget` (and not `account.budget.post`). See https://github.com/OCA/account-budgeting/blob/056fb79e2a06841b2bf11f95ee6cf890522a2567/account_budget_oca/models/account_budget.py#L114-L116

Reply:
Sorry, my previous comment is wrong.